hcl2template/parser.go

```diff
@@ -162,7 +164,7 @@ func (p *Parser) Parse(filename string, varFiles []string, argVars map[string]st
 
 	// parse var files
 	{
-		hclVarFiles, jsonVarFiles, moreDiags := GetHCL2Files(filename, hcl2VarFileExt, hcl2VarJsonFileExt)
+		hclVarFiles, jsonVarFiles, moreDiags := GetHCL2Files(filename, hcl2AutoVarFileExt, hcl2AutoVarJsonFileExt)
```

Comment:
Oh, super good catch !

Reply:
@nywilken I was referring to this change here: before this the HCL2 parse would read all `.pkr.hcl` files in the folder, now it only reads `.auto.pkr.hcl` files ( which I think is a good way to go ).

Reply:
So if anyone was mistakenly relying on this behaviour. Bang ! I say mistakenly because the docs says that only `.auto` files in the same folder will be auto parsed

Reply:
Got it, thanks.  To confirm you mean `.auto.pkrvars.hcl` , right? If so then the behavior was correct to begin with. I just renamed the variable to so that I could use `hcl2VarFileExt` for the non-auto variable file extension :smile: I'm going to merge since this this is what we want. But do let me know if I misunderstood the case for `.pkr.hcl` files.

Reply:
OOoohh right I misread the diff sorry ! Yup even better !